ansible/roles/linux-common/tasks/main.yml

```diff
@@ -2,44 +2,42 @@
 # ACL Required for become to an unprivileged user to work
 # Ensure up-to-date CA certificates
 # Use linux-generic to track linux-modules-extra (IB)
-- name: Install common packages  # noqa package-latest
-  ansible.builtin.apt:
+- name: Install common packages
+  apt:
     update_cache: true
     pkg:
-      - acl
-      - ca-certificates
-      - linux-generic
+    - acl
+    - ca-certificates
+    - linux-generic
     state: latest
 
 # Enables the ansible-init system, but does not install any default playbooks
-- name: Ensure ansible-init is configured
-  ansible.builtin.include_role:
+- include_role:
     name: azimuth_cloud.image_utils.linux_ansible_init
 
 # Enables volumes to be referred to by tag for setting up mounts in cloud-config
-- name: Ensure volumes can be mounted by tag
-  ansible.builtin.include_role:
+- include_role:
     name: linux-volumes-by-tag
 
+- include_role:
+    name: linux-nvme
```

Comment:
I wonder if we should just call this linux-extra-disks or something more generic?